packages/story-editor/src/components/canvas/displayElement.d.ts

```diff
@@ -13,9 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/**
- * External dependencies
- */
-import { createContext } from '@googleforcreators/react';
 
-export default createContext({ actions: {}, state: {} });
+import type { Element } from '@googleforcreators/elements';
```

Comment:
Added a module for `DisplayElement`, otherwise converting the provider is blocked.